### DISCUSSION TITLE
branch push 질문 드립니다

### DISCUSSION THREAD

**Discussion by Hyun-Soon** [lawnmowing-programmer/algo#295] [Q&A] [answered]
클러스터 맥 용량 문제 때문에 새로 디렉토리를 만들고, 

```
git init
git remote add origin https://github.com/lawnmowing-programmer/algo.git
```

위 명령어들로 algo 레포지토리와 연결했습니다.
이후 branch를 만들고 추가할 디렉토리(Week15)를 add, commit 후

`git push origin algo/HyunSoon/Week15`

명령어로 push를 했으나 아래 사진과 같이 pull request를 위한 버튼이 보이지 않습니다.
<img width="1220" alt="image" src="https://github.com/lawnmowing-programmer/algo/assets/66724166/a2f5347f-240d-4b98-afd5-0e1ef715f7dc">

고수님들의 해결 방법을 기다립니다..

**Comment by minjae9610** [comment, marked as answer]
![image](https://github.com/lawnmowing-programmer/algo/assets/33440010/3f088a20-b94c-4c5b-887a-cf72071668ea)
![image](https://github.com/lawnmowing-programmer/algo/assets/33440010/f0f7e8a6-b5fc-4068-ab4b-fef3f9149704)

기존에 존재하는 레포지토리가 아닌 새 레포지토리를 생성하신거라서 문제가 발생하는 경우입니다.
clone을 하시거나, 아니면 pull을 통해 메인 브랜치를 갱신하신 다음 브랜치를 생성하셔야 합니다.

**Reply by Hyun-Soon** [reply]
아 git remote -v에 등록되어 있다고 바로 연동되는게 아닌가 보네요
그래프로 보니까 직관적이네요
정말 감사합니다!
